tests/models/unit_cases_electricity/battery/src/example.py

```diff
@@ -0,0 +1,103 @@
+from mesido.esdl.esdl_mixin import ESDLMixin
```

Comment:
- I suggest to create a small test case to test that the constraints are acting as expected. While all of this is fresh in your head :-)
- Currently the charge and discharge efficiencies seem to be set to 1. It would be nice if these values are smaller than 1.0 and also different to test that what goes into the battery before the effic == values ending up in the battery after the efficiency is applied. I would check this for charging and discharging.

Reply:
The charge and discharge efficiencies are already set to a different value and 0.85 and 0.9. And this also checked in the test that these are correctly used in the calculation. 
See the last lines of test_electricty_storage.py